src/main.rs

```diff
@@ -10,6 +12,14 @@ use trin::run_trin;
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     init_tracing_logger();
     let trin_config = TrinConfig::from_cli();
+
+    // Placing this check here since TrinConfig has tests which this would fail.
+    // We also don't want to run unneeded code if we are going to panic for this anyways.
+    #[cfg(windows)]
```

Comment:
I had to read the comment a few times to understand it. Maybe something like?
```suggestion
    // Panic early on a windows build that is trying to use IPC, which is unsupported for now
    // Make sure not to panic on non-windows configurations.
    #[cfg(windows)]
```

Reply:
resolved